middleware/recover/recover.go

```diff
@@ -31,6 +31,10 @@ func New(config ...Config) fiber.Handler {
 					cfg.StackTraceHandler(c, r)
 				}
 
+				if cfg.SkipResponseError {
+					return
+				}
+
```

Comment:
It seems better to add a panic handler to customize the user behavior, the default behavior is to return err.
- refer to https://github.com/cloudwego/hertz/commit/f687dac7225c070e85c2abbf87cd4605a01402f3

Reply:
sry, there is already a StackTraceHandler and a handler can be used to do this, but there will be break changes.😞

Reply:
would be ok for me, but with fiber error handler we already have this possibility, because the error which is caught in the middleware is passed on to the error handler and this is responsible for the output or the conversion of the response.